CMakeLists.txt

```diff
@@ -259,6 +259,14 @@ opm_add_test(ebos_altidx
   DEPENDS "opmsimulators"
   LIBRARIES "opmsimulators")
 
+opm_add_test(ebos_plain
+  ONLY_COMPILE
+  DEFAULT_ENABLE_IF ${EBOS_DEBUG_EXTENSIONS_DEFAULT_ENABLE_IF}
+  SOURCES ebos/ebos_plain.cc
+  EXE_NAME ebos_plain
+  DEPENDS "opmsimulators"
+  LIBRARIES "opmsimulators")
+
```

Comment:
I assume this is a preparation for a ebos multiplexer binary since the ebos binary is kept.

Reply:
Not really: I just noticed could only be build with the dune based BS, but not with the default one. (I exclusively use the former, except occasionally for testing.) Anyway, thanks for merging.